### ISSUE TITLE
Implement WebRTC

### ISSUE THREAD

**Issue by LeonMrBonnie** [altmp/altv-issues#1379]
### Description of the problem

Add a WebRTC API available directly from client

### Desired solution for the problem

-

### Alternatives you considered

-

### Additional context

-

**Comment by xxshady** [comment]
Just wondering, with this api will we be able to do our custom synchronizations, e.g. peds, using udp?

**Comment by Yiin** [comment]
> Just wondering, with this api will we be able to do our custom synchronizations, e.g. peds, using udp?

From what I've tested, the issue with ped sync is lack of control for ragdoll/animations (as natives are pretty limited at that), not data transmission, so even if this comes out, latency might get a little bit better but that's about it, no?

**Comment by xxshady** [comment]
True, with natives you are too limited in implementing your own ped sync and thats a more big problem
